internal/input/input_js.go

```diff
@@ -91,6 +96,11 @@ func (i *Input) IsMouseButtonPressed(button MouseButton) bool {
 	return false
 }
 
+// TODO: Mouse scroll functionality is not yet implemented in js
+func (i *Input) MouseWheel() (xoff, yoff float64) {
```

Comment:
Define the counterpart in `input_mobile.go`

Reply:
Done.